### PR TITLE
Prefix header be gone!

### PR DESCRIPTION
**Pull request by alloy**
**IMPORTANT**: You need to delete the following Pods and re-run pod install: ORStackView, ARAnalytics, ARTiledImageView, DRKonamiCode, and NAMapKit.

### CODE REVIEW

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4508,6 +4512,21 @@
...
+			inputPaths = (
+			);
+			name = "Generate Compilation DB";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if type -p clang-compilation-database-tool > /dev/null 2>&1; then\n  clang-compilation-database-tool collect \"${OBJECT_FILE_DIR_normal}\" > \"${SRCROOT}/compiler_commands.json\"\nfi";
+			showEnvVarsInLog = 0;
+		};
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: what's your feels on keeping this around?

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4583,21 +4602,6 @@
...
-			inputPaths = (
-			);
-			name = "Copy Pods Resources";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-Artsy/Pods-Artsy-resources.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: did we have two going on?

File: Artsy.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: The resulting compilation DB?

File: Artsy.xcodeproj/project.pbxproj (hunk 2 above)
Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: Not sure, this was an old change, but we don’t need this at all anymore now that we have frameworks.

File: Artsy/App/ARFonts.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#import <Artsy_UIColors/UIColor+ArtsyColors.h>
```
Comment (orta) [inline Artsy/App/ARFonts.h]: Thanks - fixes #1071  #845

Comment (orta) [inline Artsy/App/ARFonts.h]: Thanks - fixes #1071  #845

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | No CHANGELOG changes made
    

  

  
    &nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

File: Artsy.xcodeproj/project.pbxproj (hunk 1 above)
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: Yeah, didn't you try this initially to see if we'd get compile times down?

**Comment by alloy** [comment]
Eh… I guess we can re-enable modules as we’re not actually going to use the caching stuff. Or how do you feel about removing the module from Aerodramus?

**Comment by alloy** [comment]
Hmm, haven’t seen those failures, will run that make task locally.

**Comment by orta** [comment]
I'm not too sure what this means:

> how you feel about removing the module from Aerodramus

If the JSON thingy is faster, keep it in, if it's slower, ditch it.

**Comment by alloy** [comment]
It was about the clang module import, but never mind, I’ve just re-enabled modules.

Not sure what the JSON thing is about, though :D

**Comment by orta** [comment]
Sorry, the clang compilation db, OK, get this rebased and it's good from me :+1:

**Comment by alloy** [comment]
Ah, well, I’m leaving it all in, but it’s disable for you all because you don’t have those tools installed. I’m disabling the generation for now, as that’s causing CI to fail and I don’t feel like debugging gCI the rest of the day :)

**Comment by alloy** [comment]
Done :+1:

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Pull request merged**